### PR TITLE
Add vision blog post

### PR DESCRIPTION
**Pull request by greysteil**
That.

### CODE REVIEW

**Comment by greysteil** [comment]
@hmarr - edits very welcome. Totally first draft.

File: blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Last week we launched [Dependabot][dependabot-homepage], a dependable robot
+who’ll keep your dependencies up-to-date for you. We've got big plans for it,
+all of which are about saving you time.
+
+### The small dream: automating the leg-work of updating
+_TL;DR: There used to be non-thinking work involved in updating a dependency.
+Dependabot already automates that away. We hope it means everyone will update
+more often._
+
+Before Dependabot, updating dependencies used to require a bunch of monotonous
```
Comment (hmarr) [inline blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md]: ✂️  "a bunch of"

File: blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+work. If you wanted to stay up-to-date you needed to check for updates, find
```
Comment (hmarr) [inline blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md]: "updates" -> "new dependency versions" ("update" sounds a bit weird just after "up-to-date")

File: blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+the relevant changelogs, and put together a pull request. It might not sounds
```
Comment (hmarr) [inline blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md]: "sounds" -> "sound"

File: blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+like much, but it was boring, and enough to stop most companies from updating
```
Comment (hmarr) [inline blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md]: boring and time consuming?

File: blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+daily.
```
Comment (hmarr) [inline blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md]: daily -> regularly?

File: blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+let us comment on the PRs we've created, reassuring you that specs are passing
+for everyone else, too. And if they're not passing, we'll be able to warn you,
+and alert the dependency maintainer, linking of an open-source repo with failing
+specs as a test case.
+
+Minor and patch releases made up X% of the last 1,000 Rubygems releases -
+making reviewing them trivial, and ensuring they're all bug-free, should save
+everyone time and make the world a little less buggy.
+
+### The pipe dream: codeshift to fix breaking changes
```
Comment (hmarr) [inline blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md]: codeshift is the name of a specific JS tool - I'd go with something more generic. "Automatic refactors for breaking changes"?

File: blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+dependency maintainers to make the changes they need to._
+
+When there's a breaking change in a dependency you rely on, it's normally time
+to dust off your keyboard and write some code. But almost everyone using the
+dependency is writing the same code. Couldn't that be automated?
+
+Some of the tools to automate a code change already exist: Facebook's
+[jscodeshift][facebook-codeshift] is already public, and there's been some
+public discussion of Google's [Rosie][google-rosie]. As a community we need to
+agree on a standard for writing codemods, and then start including them in
```
Comment (hmarr) [inline blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md]: "a standard" -> "standards" (I suspect they'll be different for each language / ecosystem)

also, "codemods" -> "automatic refactors" or "code transformations"? Again, codemod is a specific too.

File: blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+breaking changes to libraries.
+
+We hope Dependabot can help encourage the adoption of codemods by automating
```
Comment (hmarr) [inline blog/_posts/2017-05-30-a-bigger-vision-for-dependency-management.md]: codemods

**Pull request merged**